objects/client.c

```diff
@@ -1533,6 +1534,14 @@ client_manage(xcb_window_t w, xcb_get_geometry_reply_t *wgeom, xcb_get_window_at
     luaA_object_emit_signal(L, -1, "manage", 0);
     /* pop client */
     lua_pop(L, 1);
+
+    xcb_generic_error_t *error = xcb_request_check(globalconf.connection, reparent_cookie);
+    if (error != NULL) {
+        warn("Failed to manage some window, because reparenting failed.");
```

Comment:
Could we be more specific here instead of "some window"?

Reply:
The window id will not help anybody. Should `c->name` be included in the message?

I think that when this triggers, it is obvious that something is wrong (you start firefox, no window appears, but a firefox process is running). The harder part would be for people to actually find this warning. But yes, that's no excuse to produce a bad warning.

Reply:
Ok with me then.

Reply:
Ah, right, I forgot about those